apps/rpc/src/bin/server.ts

```diff
@@ -51,7 +65,14 @@ export async function createFastifyContext({ req }: CreateFastifyContextOptions)
     if (subject === undefined) {
       return createTrpcContext(null, serviceLayer)
     }
-    const editorRoles = await serviceLayer.authorizationService.getEditorRoles(serviceLayer.prisma, subject)
+
+    let editorRoles: EditorRoleSet
```

Comment:
Add a comment above why this is here please, also I think I would prefer a ternary operator here